client/rpc/Cargo.toml

```diff
@@ -24,6 +24,7 @@ sp-io = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate"
 sp-runtime = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
 sp-api = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
 sp-transaction-pool = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
+sc-transaction-pool = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", features = ["test-helpers"] }
```

Comment:
With https://github.com/paritytech/substrate/pull/9726 merged we can remove the `test-helpers` feature?